tutor4points/core/templates/tutors.html

```diff
@@ -42,9 +44,43 @@
 
 
     <div>
-        <button type ="submit" name="request" value="request" class="request-button">Request</button>
+        <button type ="submit" name="request" value="request" class="request-button" data-toggle= "modal" data-target="#tutor-request-modal">Request</button>
+    </div>
+
+    <!-- insert request form here -->
+    <div class = "modal fade" id = "tutor-request-modal">
```

Comment:
I'm not sure why, but the form only submits for the first tutor. When clicking other requests, it also only shows the first tutor.